indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/supervisor/SeekableStreamSupervisor.java

```diff
@@ -902,7 +902,9 @@ public void tryInit()
                   }
 
                   try {
+                    log.debug("Handling notice [%s] from notices queue, current notices queue size [%d]", notice.getClass().getName(), getNoticesQueueSize());
                     notice.handle();
+                    log.debug("Handled notice [%s] from notices queue, current notices queue size [%d]", notice.getClass().getName(), getNoticesQueueSize());
                   }
```

Comment:
those two debug messages are very hard to distinguish. I think we should make it more obvious to avoid confusion when debugging, or only keep one of them.

Reply:
I agree with @xvrl. It seems enough to me to keep only one of them.

Reply:
The intent for adding the 2 log statements was to get the time taken to process the notices just by looking at the 2 log lines. Now just logged the time difference instead and removed 1 of the log lines.